ui/src/conferences/components/ProceedingsAction.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List, Map } from 'immutable';
+import { Menu } from 'antd';
+
+import ExternalLink from '../../common/components/ExternalLink';
+import ActionsDropdownOrAction from '../../common/components/ActionsDropdownOrAction';
+import IconText from '../../common/components/IconText';
+import JournalInfo from '../../common/components/JournalInfo';
+
+function renderProceedingsDropdownAction(proceeding, index) {
+  const href = proceeding.getIn(['record', '$ref']);
+  const publicationInfo = proceeding.getIn(['publication_info', 0], Map());
+  return (
+    <Menu.Item key={href}>
+      <ExternalLink href={href}>
+        {publicationInfo.has('journal_title') ? (
+          <JournalInfo info={publicationInfo} />
+        ) : (
+          <span>Proceedings {index + 1}</span>
```

Comment:
should this be Proceeding or Proceedings?

Reply:
apparently `proceedingS`